account_move_line_product/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Account Move Line Product",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.0",
```

Comment:
This change has to be set in migration commit instead pre-commit one.

Reply:
Hi @rousseldenis, I am referring to OCA's standard steps. In those steps, I executed the pre-commit test locally, which indicated that the version should be 16.0.

Reply:
Yes but it says that you should commit with --no-verify :smiley: 

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#how-to